MSID_limit/compare_database_and_update.py

```diff
@@ -361,13 +362,16 @@ def download_glimmon():
     cmd = 'mv -f  ' + glimmon + ' ' + glimmon + '~'
     os.system(cmd)
 
-    with open('/home/isobe/.occpass' , 'r') as f:
-        out = f.read().strip()
+    netrc = Ska.ftp.parse_netrc()
+    if 'occweb' not in netrc:
+        raise RuntimeError('must have occweb auth in ~/.netrc')
+
+    user = netrc['occweb']['login']
+    password = netrc['occweb']['password']
 #
 #--- download the database
 #
-    cmd = 'curl -u tisobe:' + out 
-    cmd = cmd + ' https://occweb.cfa.harvard.edu/occweb/FOT/engineering/thermal/'
+    cmd = f'curl -u {user}:{password} https://occweb.cfa.harvard.edu/occweb/FOT/engineering/thermal/'
```

Comment:
This is probably not secure since it exposes the password in your processes while curl is running. Use [kadi.occweb.get_occweb_page](https://cs.github.com/sot/kadi/blob/e06ed0d554ed6711410ae1d98d689fe8f043dfbf/kadi/occweb.py#L199) instead with `binary=True`.

Reply:
it seems like Tom had a low impact change.   Do you want ot use that, or would you like to gow with this for now and change it later?